### ISSUE TITLE
Footnote overlap with page number

### ISSUE THREAD

**Issue by fc0sta** [cgogolin/quantum-journal#46]
In some case, the text in the footnote ends up overlapping with the line at the bottom of the page and with the page number. A minimal working example:

```
\documentclass[a4paper,11pt]{quantumarticle}
\usepackage{lipsum}
   
\begin{document}
\lipsum\footnote{\lipsum}

\end{document}
```

It seems to happen with long or multiple footnotes. Other document classes would automatically adjust the length of the page or move part of the footnote to the next page.

**Comment by cgogolin** [comment]
Hi! Thanks for reporting and a special thanks for the example! I don't have a fix yet, but I know the culprit. The problem is caused by `ltxgrid`. As can be seen from the following minimal example:
```
\documentclass[a4paper,11pt]{article}
\usepackage{lipsum}

\def\class@name{ltxutil}
\usepackage{ltxgrid}
\begin{document}
\lipsum\footnote{\lipsum}

\end{document}
```
In fact, `revtex4-1` has exactly the same problem:
```
\documentclass[nofootinbib,a4paper,11pt]{revtex4-1}
\usepackage{lipsum}

\begin{document}

\lipsum\footnote{\lipsum}

\end{document}
```
We need `ltxgrid` to provide the `widetext` environment, which allows wide equations in two-column mode. I fear it will be very difficult to fix this. Maybe we can simply not load `ltxgrid` in onecolumn mode at all?

**Comment by fc0sta** [comment]
Thanks!
> Maybe we can simply not load ltxgrid in onecolumn mode at all?

That should at least fix my problem.
If I understand correctly, the only purpose of `ltxgrid ` is to switch between one and two column settings, but with a couple of tests it seems buggy when the starting setting is one column. 
It should be possible to get almost all functionalities of `ltxgrid ` starting from twocolumn, so indeed it makes sense to use it only there, at least until someone fixes `ltxgrid ` for single column.

**Comment by fc0sta** [comment]
Hi,
so how do I fix that? Is there a simple way to make latex not upload ltxgrid or should I wait an updated style file?
Cheers,
Fabio. 

    Il Venerdì 1 Dicembre 2017 19:51, Christian Gogolin <notifications@github.com> ha scritto:
 

 Hi! Thanks for reporting and a special thanks for the example! I don't have a fix yet, but I know the culprit. The problem is caused by ltxgrid. As can be seen from the following minimal example:\documentclass[a4paper,11pt]{article}
\usepackage{lipsum}

\def\class@name{ltxutil}
\usepackage{ltxgrid}
\begin{document}
\lipsum\footnote{\lipsum}

\end{document}
In fact, revtex4-1 has exactly the same problem:\documentclass[nofootinbib,a4paper,11pt]{revtex4-1}
\usepackage{lipsum}

\begin{document}

\lipsum\footnote{\lipsum}

\end{document}
We need ltxgrid to provide the widetext environment, which allows wide equations in two-column mode. I fear it will be very difficult to fix this. Maybe we can simply not load ltxgrid in onecolumn mode at all?—

**Comment by cgogolin** [comment]
With b405fb8328c16d7f308d7c8b7c3dca3c00015bca I have just pushed an update that prevents the automatic loading of ltxgrid in `onecolumn` documents to the `develop` branch. With this change the minimal example
```
\documentclass[a4paper,11pt,onecolumn]{quantumarticle}
\usepackage{lipsum}

\begin{document}
\lipsum\footnote{\lipsum}

\end{document}
```
looks fine.

You can get this not yet released version of `quantumarticle.cls` from here https://github.com/quantum-journal/quantum-journal/blob/develop/quantumarticle.cls

Please let me know if it fixes your issue. Thank you!

**Comment by fc0sta** [comment]
Yes it worked. Thank you a lot!
Fabio.
 

    Il Martedì 5 Dicembre 2017 21:19, Christian Gogolin <notifications@github.com> ha scritto:
 

 With b405fb8 I have just pushed an update that prevents the automatic loading of ltxgrid in onecolumn documents to the develop branch. With this change the minimal example\documentclass[a4paper,11pt,onecolumn]{quantumarticle}
\usepackage{lipsum}

\begin{document}
\lipsum\footnote{\lipsum}

\end{document}
looks fine.You can get this not yet released version of quantumarticle.cls from here https://github.com/quantum-journal/quantum-journal/blob/develop/quantumarticle.clsPlease let me know if it fixes your issue. Thank you!—

**Comment by cgogolin** [comment]
Ok, thank you. Then I consider this issue closed through commit b405fb8.
